flink-runtime/src/main/java/org/apache/flink/runtime/state/filesystem/FsStateBackend.java

```diff
@@ -524,13 +524,17 @@ public CheckpointStorageAccess createCheckpointStorage(JobID jobId) throws IOExc
 		HeapPriorityQueueSetFactory priorityQueueSetFactory =
 			new HeapPriorityQueueSetFactory(keyGroupRange, numberOfKeyGroups, 128);
 
+		ReadableConfig config = env.getTaskManagerInfo().getConfiguration();
+		int fsReadBufferSize = config.get(CheckpointingOptions.FS_READ_BUFFER_SIZE);
```

Comment:
Do not do this here, you should use the config passed to the `configure` method. Look at how the async flag is set.

Reply:
Hi, @sjwiesman , thanks for your comments. I will do it.

Reply:
done